### PR TITLE
Adaptive - removing lives

### PR DESCRIPTION
**Pull request by alanlanglois**
[Trello](https://trello.com/c/SKEsHgUA/1679-do-not-display-lives-and-counter-everywhere-when-module-or-chapter-isconditional)
## Detailed purpose of the PR

Do not display lives( and counter) everywhere (when module or chapter isConditional)

Before: 
image

After:
image

### Testing strategy

- [ ] I ensured my code is tested: 
- [ ] I took care to mention breaking changes and extra libs if appropriate

### CODE REVIEW

File: src/layer/data/chapter-rules.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+// @flow
+import type {ChapterRule} from '@coorpacademy/player-services';
+import translations from '../../translations';
+import {getItem} from './core';
+import {CONTENT_TYPE} from './_const';
+import type {ChapterRules} from './_types';
+
+export const getChapterRulesByContent = async (
+  contentRef: string
+): Promise<Array<ChapterRule> | void> => {
```
Comment (Fausto95) [inline src/layer/data/chapter-rules.js:10]: @alanlanglois `Promise<Array<ChapterRule> | []>` ou `Promise<Array<ChapterRule> | Array<void>>`

File: src/__fixtures__/chapter-rules.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+}): ChapterRules => ({
+  _id: '59564bb2479ce0c58b7ffa94',
+  chapterRef,
+  ref,
+  meta: {
+    updatedAt: '2019-01-15T15:19:43.244Z',
+    createdAt: '2018-11-29T11:38:40.205Z'
+  },
+  rules: [
+    {
```
Comment (VincentCATILLON) [inline src/__fixtures__/chapter-rules.js:20]: in this array, please use another function named for example: `createChapterRule`, which will be called for each item you want in this array

File: src/layer/data/_types.js [hunk 3]
```diff
@@ -74,6 +74,17 @@ export type Chapter = $Exact<{|
...
 
+export type ChapterRules = $Exact<{|
+  _id: string,
+  chapterRef: string,
+  ref: string,
+  meta: {
+    updatedAt: string,
+    createdAt: string
+  },
+  rules: Array<ChapterRule>
```
Comment (VincentCATILLON) [inline src/layer/data/_types.js:85]: where is defined ChapterRule ?

Comment (alanlanglois) [reply src/layer/data/_types.js:85]: It's defined in `'@coorpacademy/player-services'` and already used line `BundledChapter` line 123

**Pull request merged**